save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/LnkUserOrganizationController.kt

```diff
@@ -193,6 +194,27 @@ class LnkUserOrganizationController(
         )
         return (exactMatchUsers + prefixUsers).map { it.toUserInfo() }
     }
+
+    /**
+     * Get not deleted organizations that are connected with current user.
+     *
+     * @param authentication
+     * @return Map where key is organization name, value is a pair of avatar and role.
+     */
+    @Suppress("TYPE_ALIAS", "UnsafeCallOnNullableType")
+    @GetMapping("/by-user/not-deleted")
+    fun getOrganizationWithRoles(
+        authentication: Authentication,
+    ): Map<String, Pair<String?, Role>> {
+        val selfId = (authentication.details as AuthenticationDetails).id
+        val user = lnkUserOrganizationService.getUserById(selfId).get()
```

Comment:
Well, `getUserId` returns an empty Optional if and only if user is not present in database.
Probably we need to process this case in order to support unauthorized users who use our api

Reply:
Fun fact is that it will never throw an exception due to authorization check. If user is unauthorized, he gets 401, otherwise we have such user in database.

Reply:
But still will add exception invocation in case of better readability